code/game/verbs/byond_membership.dm

```diff
@@ -0,0 +1,18 @@
+/client/verb/check_byond_membership()
+	set name = "Check Byond Membership"
+	set category = "OOC"
+
+	if(get_byond_membership())
+		to_chat(src, SPAN_NOTICE("Congratulations! You are a BYOND member and (depends on features of this server) might have some exclusive features!"))
```

Comment:
Would suggest calculating the perks list before this and only showing the 'you have cool features' message if they have perks on the current server, rather than awkwardly saying 'perhaps..'